cmd/rook/ceph/mds.go

```diff
@@ -31,27 +31,32 @@ var (
 	podName       string
 	filesystemID  string
 	activeStandby bool
+	keyring       string
 )
 
 var mdsCmd = &cobra.Command{
-	Use:    "mds",
+	Use:    mdsdaemon.InitCommand,
 	Short:  "Generates mds config and runs the mds daemon",
 	Hidden: true,
 }
 
 func init() {
-	mdsCmd.Flags().StringVar(&podName, "pod-name", "", "name of the pod from which the mds ID is derived")
-	mdsCmd.Flags().StringVar(&filesystemID, "filesystem-id", "", "ID of the filesystem this MDS will serve")
-	mdsCmd.Flags().BoolVar(&activeStandby, "active-standby", true, "Whether to start in active standby mode")
+	// TODO: These had to change quite a bit. Does this cause a deprecation/upgrade path change?
+	//       Or is it okay since it's a hidden command?
```

Comment:
Could you remove the spaces in the comments (`// Or is`).
There are also some other comments where I saw this, additionally could you have the first char in uppercase where applicable in your comments (e.g., line 98 in this).